exporters/trace/appoptics/appoptics_test.go

```diff
@@ -0,0 +1,43 @@
+package appoptics_test
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/contrib/exporters/trace/appoptics"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/baggage"
+	sdktrace "go.opentelemetry.io/otel/sdk/trace"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func AOSpanTest() {
```

Comment:
This is not a 'normal' test case (whose name should start with `Test` and with a `*testing.T` parameter), but rather a stub function that produces and exports the OT span.